xbmcvfs.py

```diff
@@ -0,0 +1,286 @@
+# SPDX-License-Identifier: GPL-3.0
+
+import io
+import os
+from shutil import copyfile
+
+
+class File(object):  # NOSONAR
+    def __init__(self, path, flags='r'):
+        """ File class.
+
+        :param str path:    The file or directory to open
+        :param str flags:   The flags used to open the file
+        """
+        self._file = io.open(path, flags)
+
+    def close(self):
+        """ Close the file. """
+        self._file.close()
+
+    def read(self, bytes=-1):
+        """ Read from the file.
+
+        :param int bytes:       How many bytes to read
+
+        :return: The read bytes
+        :rtype: str
+        """
+        return self._file.read(bytes)
+
+    # noinspection PyPep8Naming
+    def readBytes(self, numbytes=-1):  # NOSONAR
+        """ Read from the file.
+
+        :param int numbytes:    How many bytes to read
+
+        :return: The read bytes
+        :rtype: str
+        """
+        return self._file.read(numbytes)
+
+    def seek(self, seekBytes, iWhence=0):  # NOSONAR
```

Comment:
Should return an integer (See https://romanvm.github.io/Kodistubs/_modules/xbmcvfs.html#File.seek)

Reply:
I see, it seems that this is missing in the Kodi documentation.
-> https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/legacy/File.h#L219